src/directives/hotColumn.js

```diff
@@ -13,7 +13,7 @@
             $scope.column = {};
           }
           var optionList = {};
-          var match = options.match(/^\s*(.+)\s+in\s+(.*)\s*$/);
+          var match = options.match(/^\s*([\s\S]+?)\s+in\s+([\s\S]+?)\s*$/);
```

Comment:
`[\s\S]` can include line breaks in the match, which `.` does not.
`+?` matches one or more characters non-greedy, whereas `+` by itself is greedy.